### PR TITLE
Partially Use f string 

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?

Tried running flynt (https://github.com/ikamensh/flynt) to convert format string to fstring (only to python/ray/*.py & python/ray/util, python/ray/scripts). Unforunately, it works only for a single line... 

## Checks

- [x] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/latest/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failure rates at https://ray-travis-tracker.herokuapp.com/.
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested (please justify below)

### CODE REVIEW

File: python/ray/actor.py [hunk 1]
```diff
@@ -744,9 +744,8 @@ def __call__(self, *args, **kwargs):
                         format(item, item))
 
                 def remote(self, *args, **kwargs):
-                    logger.warning(
-                        "Actor method {} is not supported by cross language."
-                        .format(item))
+                    logger.warning(f"Actor method {item} is not "
+                                   f"supported by cross language.")
```
Comment (simon-mo) [inline python/ray/actor.py:748]: f doesn't need for this line

File: python/ray/function_manager.py [hunk 2]
```diff
@@ -274,9 +274,8 @@ def _load_function_from_local(self, job_id, function_descriptor):
             self._num_task_executions[job_id][function_id] = 0
         except Exception:
             logger.exception("Failed to load function %s.", function_name)
-            raise RuntimeError(
-                "Function {} failed to be loaded from local code.".format(
-                    function_descriptor))
+            raise RuntimeError(f"Function {function_descriptor} failed "
+                               f"to be loaded from local code.")
```
Comment (simon-mo) [inline python/ray/function_manager.py:278]: this line doesn't need f string

File: python/ray/monitor.py [hunk 3]
```diff
@@ -366,8 +365,8 @@ def run(self):
         redis_client = ray.services.create_redis_client(
             args.redis_address, password=args.redis_password)
         traceback_str = ray.utils.format_error_message(traceback.format_exc())
-        message = "The monitor failed with the following error:\n{}".format(
-            traceback_str)
+        message = (f"The monitor failed with the "
```
Comment (simon-mo) [inline python/ray/monitor.py:368]: this line doesn't need f string

**Comment by simon-mo** [comment]
checking through this PR. for multiline did you do it manually?

**Comment by rkooo567** [comment]
@simon-mo It didn't cover any multi line. If there are, that means I did manually for some reasons.

File: python/ray/services.py [hunk 4]
```diff
@@ -163,8 +163,8 @@ def find_redis_address_or_die():
             pass
     if len(redis_addresses) > 1:
         raise ConnectionError(
-            "Found multiple active Ray instances: {}. ".format(redis_addresses)
-            + "Please specify the one to connect to by setting `address`.")
+            f"Found multiple active Ray instances: {redis_addresses}. " +
```
Comment (simon-mo) [inline python/ray/services.py:166]: We don't need the `+` here

File: python/ray/services.py [hunk 5]
```diff
@@ -419,27 +419,26 @@ def start_ray_process(command,
...
-        process_type.upper())
+    valgrind_profiler_env_var = f"RAY_{process_type.upper()}_VALGRIND_PROFILER"
     if os.environ.get(valgrind_profiler_env_var) == "1":
         logger.info("Detected environment variable '%s'.",
                     valgrind_profiler_env_var)
         use_valgrind_profiler = True
-    perftools_profiler_env_var = "RAY_{}_PERFTOOLS_PROFILER".format(
-        process_type.upper())
+    perftools_profiler_env_var = (f"RAY_{process_type.upper()}"
+                                  f"_PERFTOOLS_PROFILER")
```
Comment (simon-mo) [inline python/ray/services.py:432]: no need for fstring here

File: python/ray/services.py [hunk 6]
```diff
@@ -1405,28 +1400,26 @@ def start_raylet(redis_address,
...
+        f"--maximum_startup_concurrency={maximum_startup_concurrency}",
+        f"--static_resource_list={resource_argument}",
+        f"--config_list={config_str}",
+        f"--python_worker_command={subprocess.list2cmdline(start_worker_command)}",  # noqa
+        f"--java_worker_command={subprocess.list2cmdline(java_worker_command)}",  # noqa
+        f"--redis_password={redis_password or ''}",
+        f"--temp_dir={temp_dir}",
+        f"--session_dir={session_dir}",
+        f"--metrics-agent-port={metrics_agent_port}",
+        f"--metrics_export_port={metrics_export_port}",
```
Comment (simon-mo) [inline python/ray/services.py:1422]: look at that!

File: python/ray/services.py [hunk 7]
```diff
@@ -1607,9 +1600,8 @@ def determine_plasma_store_config(object_store_memory,
                        "plasma_directory is set.")
 
     if not os.path.isdir(plasma_directory):
-        raise ValueError(
-            "The file {} does not exist or is not a directory.".format(
-                plasma_directory))
+        raise ValueError(f"The file {plasma_directory} does not "
+                         f"exist or is not a directory.")
```
Comment (simon-mo) [inline python/ray/services.py:1604]: no need for this fstring

File: python/ray/setup-dev.py [hunk 8]
```diff
@@ -37,8 +36,8 @@ def do_link(package, force=False, local_path=""):
     else:
         sudo = []
         if not os.access(os.path.dirname(package_home), os.W_OK):
-            print("You don't have write permission to {}, using sudo:".format(
-                package_home))
+            print(f"You don't have write permission "
```
Comment (simon-mo) [inline python/ray/setup-dev.py:39]: no need for this fstring

**Review by simon-mo — APPROVED** [review]
LGTM. Please fix the few minor cases around multiline f-strings

File: python/ray/function_manager.py [hunk 9]
```diff
@@ -448,8 +447,7 @@ def _load_actor_class_from_local(self, job_id,
         except Exception:
             logger.exception("Failed to load actor_class %s.", class_name)
             raise RuntimeError(
-                "Actor {} failed to be imported from local code.".format(
-                    class_name))
+                f"Actor {class_name} failed to be imported from local code.")
```
Comment (richardliaw) [inline python/ray/function_manager.py:450]: this is super redundant ;) 

```
except Exception as e:
    raise RuntimeError(f"Actor {class_name} failed to be imported from local code. Original error: {str(e)}") from None

Comment (rkooo567) [reply python/ray/function_manager.py:450]: Fixed here and the one above.

**Pull request merged**